Refinements and additional updates to the HRI task (#964)
RoboCupAtHome/RuleBook: tasks/GiveMeAHand.tex

	\item \textbf{Objects:}
		\begin{itemize}
			\item There is a box or bag with five random known objects (see \ref{rule:scenario_objects}).
			\item The arena must include visually similar delivery locations (e.g., identical bins or dishes) to create deliberate ambiguity requiring user clarification.
		\end{itemize}
\end{itemize}

\subsection*{Procedure}
\begin{enumerate}[nosep]
	\item The robot starts inside the arena, at a predefined location.
	\item The robot approaches the calling operator. The operator's location remains the same for the duration of the test.
	\item The operator takes an item out of the box and hands it to the robot. 
	\item The operator instructs the robot to place the object in a specified location in the same room.

> 	\item The robot approaches the delivery area where there is intended location ambiguity and requests clarification.
> 	\item The robot waits for user confirmation before proceeding with object placement.

Reviewer: This section now feel to constrained  :sweat_smile: Maybe remove these?

```suggestion

```

		\item The human must complete the handover by placing the object into the robot's manipulator; the robot must remain stationary and must not initiate physical contact during this phase.
		\item Explicit operator commands during the object handover—such as \textit{“Please, place the item in my gripper and wait until I close it in three, two, one”}—will be considered \textit{Deus ex Machina}. However, the robot may notify the user before closing its gripper after detecting that the object is within its grasping range; for example, \textit{“I have detected that the object is in my hand. I will close my gripper in three seconds—please be careful.”}
	\end{itemize}
	\item \textbf{Delivery:}
	\begin{itemize}[nosep]
		\item Delivery instructions may include phrases like: \textit{"Place this apple on that plate"} or \textit{"Put this can in the trash bin."} The operator gives instructions using both verbal commands and gestures (e.g., \textit{"Place this item there} \textit{(pointing), next to the flower vase"}).
		\item At any moment, the robot may ask questions to confirm information, request clarification, or ask the operator to repeat the instruction.

> 		\item Since the target locations are intentionally ambiguous, the robot must request clarification from the operator before proceeding. This must be done through natural interaction, such as verbal questions or multimodal cues (e.g., pointing, asking “Which one?”). The robot must not guess or place the object without confirmation.

Reviewer: ```suggestion
		\item Since the target locations are intentionally ambiguous, the robot can request clarification from the operator before proceeding. This should be done through natural interaction, such as verbal questions or multimodal cues (e.g., pointing, asking “Which one?”). 
```

		\item Full points for a correct delivery will only be awarded if the object is placed inside the designated container (e.g., bins) or within a few centimeters of the intended location (e.g., plates or surfaces).
		\item In this task, natural human-robot interaction is the primary focus. Before engaging with the operator to request clarification, the robot must first capture their attention — for example, through visual signals, gestures, or sound cues — rather than relying on unattended interactions (e.g., talking without addressing a specific person). This is particularly important when the operator and the robot are not in close proximity; for instance, the operator might be occupied retrieving the next object or speaking with the referee.
	\end{itemize}
\end{enumerate}

\subsection*{Instructions:}
\subsubsection*{Referee instructions}

> Timer starts when the operator begins calling to the robot.

Reviewer: I always find that starting inside the arena makes it more chaotic. Is there a rationale for not just starting outside?

Author: Good point. It can start outside the arena and define the room two hours before the test

Reviewer 2: Yes, I prefer tasks starting at outside the entrance
